DockStat.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+##########################################################
+##                                                      ##
+#    Helper script to show running Docker containers     #
+#             and related Docker resources               #
+##                                                      ##
+##########################################################
+
+get_load_avg() {
+  # "system_profiler" exists only on MacOS, if it's not here, then run linux style command for
+  # load avg.  Otherwise use MacOS style command
+  if ! command -v "system_profiler" &>/dev/null; then
+    awk '{ print  $1 " " $2 " " $3 }' </proc/loadavg
```

Comment:
how does this script behave if awk is not available?

Reply:
poorly! we can be defensive here.  see 281286a for such a defense!

Reply:
@ch4lox - you'd suggested using `docker ps --format '{{json .}}' ` along with  `jq`. I love this!  However, it was gonna be a lot more work and add another item to the fix in 281286a. Given this works "out of the box" with out any extras, Imma pass on it now.

Thanks @firebus  for letting me use your PR as code review so I can give @ch4lox feedback on his Signal as feedback from the other month :upside_down_face: